### ISSUE TITLE
When will the code updated？

### ISSUE THREAD

**Issue by Hongbo123467** [Ricky-Developer/MapDistill#1]
As a beginner, I am looking forward to following this solid work and the handsome author.

**Comment by BUAA-RickyLi** [comment]
Thank you for your kind words and interest in our work! I'm glad to hear that you're excited to follow along. We are preparing the codes and may release around August 2024 or sooner.

**Comment by LIvanoff** [comment]
I hope you will share the code and checkpoint of the lidar model)

**Comment by LeningWang** [comment]
> As a beginner, I am looking forward to following this solid work and the handsome author.

me too! excellent work!!!

**Comment by ZJWang9928** [comment]
Hi, when would the code be released?

**Comment by BUAA-RickyLi** [comment]
> Hi, when would the code be released?

Hi, Zijie. I'm sorry that I've been occupied with a lot of other work recently, which has taken up much of my time. The code and related documents will be released this weekend. Thank you for your attention. If you have any other questions, feel free to add my WeChat account: l13701391655.

**Comment by isht7** [comment]
Is the code released now?

**Comment by BUAA-RickyLi** [comment]
> Is the code released now?

yes, If you have any other questions, feel free to contact with me.
